_content/av/geometry-study_kawae-yuki.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Geometry Study: Pattern Explorations"
+subcat: film
+authors:
+  - "Yuki Kawae"
+external_url: "https://www.youtube.com/watch?v=yj2L07KwL-c"
+course: japanese
+tags:
+  - bart
+year: 2022
+month: feb
+minutes: 7
+---
+
+In this video, artist and designer Yuki Kawae showcases a handful of Zen garden patterns in sand.
+
+To learn more about Yuki Kawae and his approach to Zen gardens see (here)[https://whitewall.art/art/yuki-kawae-eases-the-mind-with-simple-tools-materials-and-patterns/].
```

Comment:
you got the format backwards. See the preview: https://deploy-preview-528--obu.netlify.app/content/av/geometry-study_kawae-yuki

You need to flip the []s and ()s 😊

Also, for the link text I like a little more description than just "here":

```md
To learn more about Yuki Kawae and
his approach to Zen gardens and art, see
[his 2021 interview with Whitewall](https://whitewall.art/art/yuki-kawae-eases-the-mind-with-simple-tools-materials-and-patterns/).
```